packages/import-notation/index.js

```diff
@@ -58,6 +58,7 @@ function parse(importString, ctx) {
             if(!main.block) {
                 main.block = ctx.block;
                 main.elem || ctx.elem && (main.elem = ctx.elem);
+                acc.add(main);
```

Comment:
need to fix jsdoc "[ctx] - entity to restore `block` part" > "[ctx] - entity to restore `block`/`elem` parts"

Reply:
We call it scope in the rest modules. Probably better to rename it here too.